RELEASE.md

```diff
@@ -0,0 +1,19 @@
+# Release
+
+osd-network-verifier will follow semantic versioning
+
+## Considerations around changes
+
+These changes to these don't automatically mean a change is a breaking or significant change, but should be taken into consideration:
+
+* The various input verifier structs in [./pkg/verifier/package_verifier.go](./pkg/verifier/package_verifier.go) are exported and consumed downstream. Breaking changes to that input struct should be considered breaking changes for osd-network-verifier.
+* New AMIs in [./pkg/verifier/aws/aws_verifier.go](./pkg/verifier/aws/aws_verifier.go), especially as the result of security fixes.
+* New cloud IAM requirements/new cloud infrastructure to provision
+
+## Testing changes
+
+For now, this is mostly manual. It's important to validate that these scenarios are working before making a new release:
+
+* [./integration](./integration/) has steps to run the egress test against an AWS account to test that the osd-network-verifier is able to run using the pre-baked image on the default AMI.
+* egress test in AWS with a cluster-wide proxy
+* egress test on GCP
```

Comment:
Are these two bullets TODO's? If not, it's unclear what this list is describing.

Reply:
Oh I was referring to running `./osd-network-verifier egress` in certain scenarios

Reply:
Ahhh okay. Re-reading the lines above it, this makes much more sense now